### PR TITLE
Update work and about me section

### PR DESCRIPTION
**Pull request by abdulrahmanshr75**
- added the linters
- configured the site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            <a href="#"
+              ><img class="icons" src="images/github.png" alt="github"
+            /></a>
+            <a href="#"
+              ><img class="icons" src="images/angellist.png" alt="angellist"
+            /></a>
+          </div>
+        </section>
+      </div>
+      <div class="second-container">
```
Comment (keneogu) [inline index.html]: Let's make use of the `<section>` semantic tag instead of the `<div>` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        <section>
+          <div class="second-container">
+            <div class="content">
+              <h2 class="heading">About me</h2>
+              <p class="p">
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don’t hestiate to contact me.
+              </p>
+              <a class="button2" href="#">Get My Resume</a>
```
Comment (keneogu) [inline index.html:162]: Add styling  for all your buttons with the `hover` `pressed` properties

File: index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+              <img src="images/Project-1.jpg" alt="Project-1" />
+              <div class="bottom">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="social-icons">
+                  <li class="languagesdef">Ruby on rails</li>
+                  <li class="languagesdef">css</li>
+                  <li class="languagesdef">Javascript</li>
+                  <li class="languagesdef">html</li>
+                </ul>
+                <a class="button" href="#">See Project</a>
```
Comment (keneogu) [inline index.html:73]: Add a styling for the buttons with `hover` `pressed` properties

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @abdulrahmanshr75 , 

Good job so far!

- Linters are working
- well structured codes

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by rustam-tolipov — APPROVED** [review]
## Hi @abdulrahmanshr75  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**